platform/core/src/utils/combineFrameInstance.ts

```diff
@@ -0,0 +1,27 @@
+import Combine from './Combine';
+
+/**
+ * Combine the Per instance frame data, the shared frame data
+ * and the root data objects.
+ */
+const combineFrameInstance = (frame, instance) => {
+  const {
+    PerFrameFunctionalGroupsSequence,
+    SharedFunctionalGroupsSequence,
+  } = instance;
+  if (!PerFrameFunctionalGroupsSequence || frame === undefined) return instance;
+  const shared = Object.values(SharedFunctionalGroupsSequence[0])
+    .map(it => it[0])
+    .filter(it => it !== undefined && typeof it === 'object');
```

Comment:
Technically, the non-object values should be merged too, but as the only thing tthere is private creators, and the private instance on that gets munged out of the return object, I don't think we really need them.